src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/random/RandomUtilities.kt

```diff
@@ -31,9 +31,15 @@ internal fun fullyPopulatedRecall(recallId: RecallId = ::RecallId.random()): Rec
         document ->
         document.copy(
           recallId = recallId.value,
-          // ensure document version is valid verus category
+          // ensure document version is valid versus category
           version = if (document.category.versioned) Random.nextInt() else null
         )
+      }.toSet(),
+      missingDocumentsRecords = recall.missingDocumentsRecords.map { mdr ->
+        mdr.copy(
+          recallId = recallId.value,
+          emailId = recall.documents.random().id
+        )
```

Comment:
Have you seen any instability in tests using `fullyPopulatedRecall`?   The comment `// ensure document version is valid versus category` and code for that is something I needed to add for that with versioned documents.